rollout.py

```diff
@@ -206,6 +218,8 @@ def _main():
             help='rollout start (e.g. "10 am")')
     add.add_argument('duration', metavar='duration-hours', type=int,
             help='rollout duration (e.g. "48")')
+    add.add_argument('--replace', action='store_true',
```

Comment:
Hmm, wonder if this should be a separate `edit` subcommand instead.

Or maybe simpler if we just rename the command e.g. `set` and it either adds or updates the existing one.